### ISSUE TITLE
Highlight selected window title (Enhancement for only-keyboard usage)

### ISSUE THREAD

**Issue by goyou** [spyder-ide/spyder#2344]
Hello,

I am using a lot the Ctrl+E and Ctrl+C shortcuts for switching from editor to console and reversely.
That way I use spyder without the mouse usually and it's working well.
But the only graphical information you have for knowing which of these windows is currently active is the text cursor.

The active window title ('Editor - ...' or 'Console') should be highlighted with darker or colored background.

**Comment by goanpeca** [comment]
Indeed, it would make it more evident, we will see what we can do about this

**Comment by ccordoba12** [comment]
After 2.3.3 (I think), this information is visible in

    View > Panes

I think that's enough, don't you think?

**Comment by goanpeca** [comment]
The think is that looking at the question, the problem seems to be that besides a blinking cursor on one or the other, there is no direct self evident way (without having to use mouse or keyboard) to know which widget is focused...

Maybe we can change the way the tab that is focused looks. Spyder right now.... Who has focus?

![image](https://cloud.githubusercontent.com/assets/3627835/7262331/05376476-e879-11e4-983e-cba228d69849.png)

To give an example of a random idea (I just inverted the color of the tab in an image editor...). Who has focus?

![image](https://cloud.githubusercontent.com/assets/3627835/7262393/735ca876-e879-11e4-8305-f9d9d59a8842.png)
https://github.com/spyder-ide/spyder/issues/2344#

**Comment by goyou** [comment]
I've made another image to show how it is in matlab : 
![05376476-e879-11e4-983e-cba228d698492](https://cloud.githubusercontent.com/assets/11740865/7269718/79e6d90c-e8d6-11e4-9d4e-7ae84317add7.png)

But maybe it's easier to fix just by bolding the windows title. I think it doesn't break the look and it's evident enough : 
![05376476-e879-11e4-983e-cba228d698493](https://cloud.githubusercontent.com/assets/11740865/7269764/f56344bc-e8d6-11e4-9a79-4ff0a2a66cc7.png)

**Comment by goyou** [comment]
My advice, at first, was just to darker a little the title background : 
![05376476-e879-11e4-983e-cba228d698494](https://cloud.githubusercontent.com/assets/11740865/7269896/f75a6358-e8d7-11e4-944c-65b1837b117b.png)
That's the way I think it fits the best.

**Comment by goanpeca** [comment]
Yep, definitely doable, and goes in hand with a change on the look of the dockwidget titile bars.

@ccordoba12, the PR #2288 that I closed could also address this issue. Now that some more benefits arise it makes sense to reopen it, and decide what would be the best course of action, we could add a light cerulean blue (or whatever color scheme we choose) to be the background of the title bar.

**Comment by ccordoba12** [comment]
I find this suggestion of highlighting the active plugin a bit ugly. I'll have to see if it can be done in a more beautiful way :-)

**Comment by ccordoba12** [comment]
I just saw how things work in Matlab, and I have to agree this is a very good suggestion to improve usability.

Let's wait for @goanpeca to implement it ;-)

**Comment by goanpeca** [comment]
Can you post a acreenshot of how it looks in matlab¿

**Comment by Nodd** [comment]
You have it above: https://github.com/spyder-ide/spyder/issues/2344#issuecomment-95068569

**Comment by goanpeca** [comment]
I mean a screenshot of matlab

**Comment by Nodd** [comment]
Doh, I was certain that it was from matlab, the look is really similar !

**Comment by goyou** [comment]
https://duckduckgo.com/?q=matlab+interface&t=canonical&iax=1&ia=images&iai=http%3A%2F%2Fetools.fernuni.ch%2Fmatlab%2Fmatlab2%2Fen%2Fimage%2Fgui.png

**Comment by goyou** [comment]
why not just adding a border to the title : 
![05376476-e879-11e4-983e-cba228d698495](https://cloud.githubusercontent.com/assets/11740865/7344003/33980156-ecc8-11e4-8200-beb6d6ef3886.png)
or to the window : 
![05376476-e879-11e4-983e-cba228d698496](https://cloud.githubusercontent.com/assets/11740865/7344010/3d8fb47e-ecc8-11e4-9b48-2902718a5817.png)
or both : 
![05376476-e879-11e4-983e-cba228d698497](https://cloud.githubusercontent.com/assets/11740865/7344013/455deed2-ecc8-11e4-824a-b896b9840c39.png)

Ps : My "hand-made" image will always be uglier than built-in Qt rendering.

**Comment by Nodd** [comment]
I find borders of the same colors not obvious enough.
Keep in mind that there is a discussion to give the possibility to hide the tockwidgets titles (see https://github.com/spyder-ide/spyder/pull/2345#issuecomment-96438578) so the highligh should also be visible without them. This rules out your first suggestion.

**Comment by goyou** [comment]
Another possibility : change the background colour of the whole widget/pane.
This can also be an option where the user chooses if he wants that behaviour and chooses the colour of the background of the focused pane.

Let me summarize the possibilities until now : 
1 - highlight one tab of the focused pane
2 - add a border to the focused pane
3 - colouring differently the background of the pane
and (useless if pane title if removed) : 
4 - change the colour of the title background
5 - add a border to title
6 - change title font

**Comment by ccordoba12** [comment]
The selected pane is highlighted now, yes, but it's barely visible in some screen resolutions. So, I think we can improve that.
